### PR TITLE
Fix API documentation link, rename `get_owned_ids` to `get_unspents

### PR DESCRIPTION
**Pull request by zack-bitcoin**
This solves #603, and solves #577.

### CODE REVIEW

**Comment by zack-bitcoin** [comment]
My mistake. This reduces test coverage. Don't accept it.

**Comment by vrde** [comment]
Hi Zack, thanks for your PR. I've edited the comment so GitHub can automatically close issues when we merge it.

Before we can merge this pull request, which may contain your intellectual property in the form of copyright or patents, our lawyers say we need you or your organization to agree to one of our contributor agreements. If you are contributing on behalf of yourself (and not on behalf of your employer or another organization you are part of) then you should:

- Go to: https://www.bigchaindb.com/cla/
- Read the Individual Contributor Agreement
- Fill in the form "For Individuals"
- Check the box to agree
- Click the SEND button

If you're contributing as an employee, and/or you want all employees of your employing organization to be covered by our contributor agreement, then someone in your organization with the authority to enter agreements on behalf of all employees must do the following:

- Go to: https://www.bigchaindb.com/cla/
- Read the Entity Contributor Agreement
- Fill in the form "For Organizations”
- Check the box to agree
- Click the SEND button

We will email you (or your employer) with further instructions.

**Comment by zack-bitcoin** [comment]
I filled out the form as you asked.
This pull reduces coverage, and I encourage you to not accept it.

**Comment by vrde** [comment]
> This pull reduces coverage, and I encourage you to not accept it.

I don't understand why it reduces code coverage, you just renamed a method and changed a link, so it should be the same as before.

**Comment by vrde** [comment]
Might be related to http://status.codecov.io/ ?

**Comment by TimDaub** [comment]
Would be cool if we could investigate why coverage decreased, fix it and then merge.
Otherwise, we can also just simply merge as the PR addresses the issues appropriately, plus 0.15% code coverage decrease is tolerable.

**Comment by ttmc** [comment]
I'm going to reopen this PR because I don't think a minor unexpected dip in code coverage warrants closing it.

@zack-bitcoin Please see the email where I gave you a long random string to paste in these comments as evidence that you're the same person who signed the Contributor License Agreement.

File: bigchaindb/processes.py [hunk 1]
```diff
@@ -5,17 +5,16 @@
...
-
 logger = logging.getLogger(__name__)
 
 BANNER = """
 ****************************************************************************
 *                                                                          *
 *   Initialization complete. BigchainDB is ready and waiting for events.   *
 *   You can send events through the API documented at:                     *
-*    - http://docs.bigchaindb.apiary.io/                                   *
+* https://bigchaindb.readthedocs.io/en/latest/drivers-clients/http-client-server-api.html *
```
Comment (sohkai) [inline bigchaindb/processes.py]: Note that this will have to change in regards to https://github.com/bigchaindb/bigchaindb/pull/650.

Comment (ttmc) [inline bigchaindb/processes.py]: Thanks @sohkai - the new URL of that page is:

https://bigchaindb.readthedocs.io/projects/server/en/latest/drivers-clients/http-client-server-api.html

**Comment by ttmc** [comment]
I'm going to close this PR because it should probably be resolved in two separate PRs:

1. A PR to fix the link to the HTTP API documentation (currently links to Apiary). See issues #577 and #671 
2. A PR to rename `get_owned_ids`. This should be done around the same time as the renaming of `owners_before` and `owners_after`, maybe in the same PR, see issue #626

**Pull request closed without merging**